docs/cookbook/makeExpandedTemporal.mjs

```diff
@@ -76,8 +76,10 @@ class ExpandedPlainDate extends Temporal.PlainDate {
     if (item instanceof Temporal.PlainDate) {
       return this._convert(item, BigInt(item.year));
     }
-    const calendar = Temporal.Calendar.from('iso8601');
-    return calendar.dateFromFields(item, undefined, this);
+    const expandedYear = BigInt(item.year);
+    const isoYear = isLeapYear(expandedYear) ? 1972 : 1970;
+    const result = super.from({ ...item, year: isoYear });
+    return this._convert(result, expandedYear);
```

Comment:
It seems like it ought to have been wrong ever since PrepareTemporalFields started coercing the fields to the expected types. But the line `const date = ExpandedTemporal.PlainDate.from({ year: 635427810, month: 2, day: 2 });` still functioned correctly up until the changes made in this PR. Anyway, it's correct now :shrug: